sample-laravel/app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Pagination\Paginator;
+use App\Models\User;
+
+class RegisterController extends Controller
+{
+    public function checkValidate(Request $request) 
+    {
+        $request->validate([
+            'mail_address' => 'required|email|unique:users|max:100',
+            'password' => 'required|alpha|max:255',
+            'confirm_password' => 'required|alpha|max:255|same:password',
+            'name' => 'required|max:255',
+            'address' => 'max:255',
+            'phone' => 'numeric|digits_between:1,15',
+        ]);
+        $data = new User();
+        $query = $data->addUser($request);     
```

Comment:
Dùng trực tiếp eloquent , hàm create là được, đọc docs link dưới:
https://laravel.com/docs/9.x/eloquent#inserting-and-updating-models